Mage.Sets/src/mage/cards/m/MariTheKillingQuill.java

```diff
@@ -66,12 +66,14 @@ public MariTheKillingQuill(UUID ownerId, CardSetInfo setInfo) {
         GainAbilityControlledEffect gainDeathTouchEffect = new GainAbilityControlledEffect(DeathtouchAbility.getInstance(), Duration.WhileOnBattlefield, filter);
         Ability mainAbility = new SimpleStaticAbility(Zone.BATTLEFIELD, gainDeathTouchEffect);
 
+        String abilityText = "Whenever this creature deals combat damage to a player, " +
+                "you may remove a hit counter from a card that player owns in exile. " +
+                "If you do, draw a card and create two Treasure tokens.";
+
         // NOTE: Optional part is handled inside the effect
-        Ability dealsDamageAbility = new DealsCombatDamageToAPlayerTriggeredAbility(new MariTheKillingQuillDealsDamageEffect(), false, true);
+        Ability dealsDamageAbility = new DealsCombatDamageToAPlayerTriggeredAbility(new MariTheKillingQuillDealsDamageEffect(), false, true).setTriggerPhrase(abilityText);
```

Comment:
It's not correct to include effect text in setTriggerPhrase.

Instead, you should set staticText within the effect constructor:

```java
    MariTheKillingQuillDealsDamageEffect() {
        super(Outcome.Benefit);
        staticText =  "you may remove a hit counter from a card that player owns in exile. " +
                "If you do, draw a card and create two Treasure tokens.";
    }
```

Reply:
Ah, fantastic! I figured there had to be a better way, so thank you for the call out. I've gone ahead and made that change.